media/src/main/java/com/tealium/media/Media.kt

```diff
@@ -157,17 +190,45 @@ class Media(private val context: TealiumContext,
      */
     fun updatePlayerState(state: PlayerState) = currentSession?.updatePlayerState(state)
 
+    override fun onActivityPaused(activity: Activity?) {
+        // background media tracking is not enabled, start timer to end session
+        if (!backgroundSessionTrackingEnabled && currentSession?.isBackgrounded == false) {
+            currentSession?.isBackgrounded = true
+            endSessionTimer = Timer("End Media", true).apply {
```

Comment:
couple of thoughts on this.
1. this method gets called when any activity is paused, even to move to a new activity; which doesn't necessarily mean `backgrounded` - the `BatchingValidator` uses a slightly different approach of keeping track of the activities to determine whether it's in the background or not. 
2. Since a Timer is associated with a new thread each time, this would be creating a new thread on each new activity. Could we rework to reuse the timer and cancel/regenerate the TimerTask each time instead?